### ISSUE TITLE
The accuracy always be 0.5

### ISSUE THREAD

**Issue by gamersover** [uvipen/Character-level-cnn-pytorch#1]
I have trained **yelp_review_polarity_small** model，and get accuracy 0.5，the output like that
```
Epoch: 1/20, Iteration: 1751/2188, Lr: 0.001, Loss: 0.6997024416923523, Accuracy: 0.4375
Epoch: 1/20, Iteration: 1801/2188, Lr: 0.001, Loss: 0.7156904935836792, Accuracy: 0.4765625
Epoch: 1/20, Iteration: 1851/2188, Lr: 0.001, Loss: 0.6968995928764343, Accuracy: 0.51953125
Epoch: 1/20, Iteration: 1901/2188, Lr: 0.001, Loss: 0.6940716505050659, Accuracy: 0.50390625
Epoch: 1/20, Iteration: 1951/2188, Lr: 0.001, Loss: 0.6941938400268555, Accuracy: 0.4609375
Epoch: 1/20, Iteration: 2001/2188, Lr: 0.001, Loss: 0.694003701210022, Accuracy: 0.50390625
Epoch: 1/20, Iteration: 2051/2188, Lr: 0.001, Loss: 0.6908363103866577, Accuracy: 0.52734375
Epoch: 1/20, Iteration: 2101/2188, Lr: 0.001, Loss: 0.7009902596473694, Accuracy: 0.46484375
Epoch: 1/20, Iteration: 2151/2188, Lr: 0.001, Loss: 0.6958677172660828, Accuracy: 0.484375
Epoch: 1/20, Lr: 0.001, Loss: 0.6933960318565369, Accuracy: 0.5
Epoch: 2/20, Iteration: 1/2188, Lr: 0.001, Loss: 0.6975014209747314, Accuracy: 0.4296875
Epoch: 2/20, Iteration: 51/2188, Lr: 0.001, Loss: 0.6942330002784729, Accuracy: 0.50390625
Epoch: 2/20, Iteration: 101/2188, Lr: 0.001, Loss: 0.6899641156196594, Accuracy: 0.5234375
```

And log.txt file 
```
Model's parameters: {'alphabet': 'abcdefghijklmnopqrstuvwxyz0123456789,;.!?:\'"/\\|_@#$%^&*~`+-=<>()[]{}', 'max_length': 1014, 'feature': 'small', 'optimizer': 'adam', 'batch
_size': 256, 'num_epochs': 20, 'lr': 0.001, 'dataset': 'yelp_review_polarity', 'es_min_delta': 0.0, 'es_patience': 3, 'input': 'input/yelp_review_polarity_csv', 'output': 'ou
tput/yelp_review_polarity_small', 'log_path': 'tensorboard/char-cnn'}Epoch: 1/20
Test loss: 0.6933960318565369 Test accuracy: 0.5
Test confusion matrix:
[[    0 19000]
 [    0 19000]]
```

I don't know why, this is diffirent with your result, please help me, thank you.
